x-pack/plugins/enterprise_search/public/applications/app_search/index.tsx

```diff
@@ -113,29 +112,15 @@ export const AppSearchNav: React.FC<IAppSearchNavProps> = ({ subNav }) => {
   return (
     <SideNav product={APP_SEARCH_PLUGIN}>
       <SideNavLink to={ENGINES_PATH} subNav={subNav} isRoot>
-        {i18n.translate('xpack.enterpriseSearch.appSearch.nav.engines', {
-          defaultMessage: 'Engines',
-        })}
+        {ENGINES_TITLE}
       </SideNavLink>
-      {canViewSettings && (
-        <SideNavLink to={SETTINGS_PATH}>
-          {i18n.translate('xpack.enterpriseSearch.appSearch.nav.settings', {
-            defaultMessage: 'Settings',
-          })}
-        </SideNavLink>
-      )}
+      {canViewSettings && <SideNavLink to={SETTINGS_PATH}>{SETTINGS_TITLE}</SideNavLink>}
```

Comment:
Sometimes I wish Prettier were a _little_ less opinionated / let us leave JSX as multi-line if we wanted to :dead_inside:

Reply:
But overall I still love Prettier haha